schema/layer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "/layer",
+  "type": "object",
+  "properties": {
+    "layer": { "type": "string" },
+    "count": { "type": "number" },
+    "geometry": { "type": "string" },
```

Comment:
Mapbox-geostats currently also generates `LineString` instead of `Line`, so I'm going to leave Tippecanoe also saying `LineString` unless there is a corresponding geostats change.